### PR TITLE
feat: use fully qualified registry names

### LINKED ISSUE #503: Feature: Use fully qualified registry names in Dockerfiles
### Motivation:
Currently, the service Dockerfiles use unqualified image names, which is generally not an issue with Docker. However, I recently tried to build an image with Podman/Buildah and that failed the first time I tried to build it (without me changing the Dockerfile or `/etc/containers/registries.conf`) with "image unqualified".

### Description:
I expect to be able to build the images with Podman (`libbuildah`).

### Exit criteria:
- [ ] service image Dockerfiles contain fully qualified registry names
- [ ] consequently, service images build with Podman

### PR DESCRIPTION
**Pull request by wULLSnpAXbWZGYDYyhWTKKspEQoaYxXyhoisqHf**
as title, use fully qualified registry names in service Dockerfiles.
fixes #503.

### CODE REVIEW

**Comment by MarioUhrik** [comment]
I think we might as well wait for @MiroslavRepka to finish his work, which will deprecate this PR - correct me if I'm wrong

**Comment by MiroslavRepka** [comment]
I don't think this regards a registry we use, rather than a tool people will use to build an image. It is explained in the linked issue (#503).

 It is nothing we have to worry about for now since we use `docker build`, however, if we switch to other tools like a `podman` the issue becomes relevant.

File: services/kube-eleven/Dockerfile [hunk 1]
```diff
@@ -39,7 +39,7 @@ WORKDIR /go/services/kube-eleven/server
 #Compile the golang code, CGO_ENABLE=0 removes cross compile dependencies
 RUN CGO_ENABLED=0 go build
 
-FROM alpine
+FROM docker.io/library/alpine
```
Comment (MiroslavRepka) [inline services/kube-eleven/Dockerfile:42]: Can we pin the version of the `alpine` image to some reasonable version while we are at it?

File: services/terraformer/Dockerfile [hunk 2]
```diff
@@ -40,7 +40,7 @@ WORKDIR /go/services/terraformer/server
 #Compile the golang code to /out, CGO_ENABLE=0 removes cross compile dependencies
 RUN CGO_ENABLED=0 go build
 
-FROM alpine
+FROM docker.io/library/alpine
```
Comment (MiroslavRepka) [inline services/terraformer/Dockerfile:43]: ditto

**Review by MiroslavRepka — CHANGES_REQUESTED** [review]
LGTM, please see my comments.

**Comment by MarioUhrik** [comment]
Sorry guys, I completely misunderstood this PR/issue.
I agree we can merge this.

I mistakenly thought this is about the `image:` field inside the Deployment manifest.

FYI @wULLSnpAXbWZGYDYyhWTKKspEQoaYxXyhoisqHf

**Comment by wULLSnpAXbWZGYDYyhWTKKspEQoaYxXyhoisqHf** [comment]
no worries there :)) @MarioUhrik 


> I don't think this regards a registry we use, rather than a tool people will use to build an image. It is explained in the linked issue (#503).
> 
> It is nothing we have to worry about for now since we use `docker build`, however, if we switch to other tools like a `podman` the issue becomes relevant.
> 
> Afaik, we can merge this, it will not break anything, since the pipeline built images successfully.

correct, thanks, yes, this concerns the paths of the (base) images used inside the `service/x` Dockerfiles.


> I agree we can proceed with this.

awesome :rocket:

File: services/kube-eleven/Dockerfile (hunk 1 above)
Comment (wULLSnpAXbWZGYDYyhWTKKspEQoaYxXyhoisqHf) [reply services/kube-eleven/Dockerfile:42]: absolutely, we could do `3.17` or `3.17.1`, depending on for how long we want to call it a day (minor version tag is more precise, although it still isn't a hash).

**Pull request closed without merging**